composer.json

```
@@ -9,13 +9,14 @@
     "license": "MIT",
     "require": {
         "php": "^8.0",
-        "sylius/sylius": "^1.12",
+        "sylius/sylius": "~1.12.15",
```

As this is a template for plugins, imo it is good to have a wider dependency
```suggestion
        "sylius/sylius": "~1.12.0",
```